doc/k8s/sqlflow-server.yaml

```diff
@@ -19,7 +19,7 @@ spec:
         name: sqlflow-server
         imagePullPolicy: Always
         command: ["sqlflowserver"]
-        args: ["--enable-session"]
```

Comment:
Please remove this line as the comment: https://github.com/sql-machine-learning/sqlflow/pull/1632#issuecomment-571648841

Reply:
Thanks @Yancey1989 .  I am doing this quick change and merge.